src/main/java/stsjorbsmod/patches/LegendaryPatch.java

```diff
@@ -32,7 +32,9 @@
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
+import static stsjorbsmod.JorbsMod.JorbsCardTags.DECK_OF_TRIALS;
```

Comment:
I'm confused about why the changes to `LegendaryPatch` would be necessary; I'd have expected all the deck of trials cards to just be `SPECIAL` rarity and to never appear in the first place except via the `DeckOfTrials`

Reply:
Admittedly, I don't remember; and probably didn't have a particular reason. 🤦‍♂